morpheushelper/util.py

```diff
@@ -50,7 +60,7 @@ async def set_prefix(new_prefix: str):
 
 
 async def check_wastebasket(
-    message: Message, member: Member, emoji: PartialEmoji, footer: str, permission: Permission
+        message: Message, member: Member, emoji: PartialEmoji, footer: str, permission: Permission
```

Comment:
it should only be 4 space for Indentation [PEP8](https://www.python.org/dev/peps/pep-0008/#indentation)